pkg/config/config_template.yaml

```diff
@@ -241,6 +238,36 @@ api_key:
 #
 # log_level: info
 # log_file: /var/log/datadog/agent.log
+
+# Set to 'yes' to output logs in JSON format
+# log_format_json: no
+
+# Set to 'no' to disable logging to stdout
+# log_to_console: yes
+
+# Set to 'yes' to disable logging to the log file
+# disable_file_logging: no
+
+# Set to 'yes' to enable logging to syslog.
+#
+# log_to_syslog: no
+#
+# If 'syslog_uri' is left undefined/empty, a local domain socket connection will be attempted
+#
+# syslog_uri:
+#
+# Set to 'yes' to output in an RFC 5424-compliant format
+#
+# syslog_rfc: no
+#
+# If connecting through a TCP socket, set to 'yes' to enable TLS
+#
+# syslog_tls: no
+#
+# If 'syslog_tls' is enabled, you can set your pem-encoded root ca certs here
+#
+# syslog_pem:
```

Comment:
you're 100% right... this actually sucks 🙈  
These should probably be paths to PEM certificates, and we'll then have to load those into memory (into `x509.Certificate`s and append those to the logger). I can fix the implementation... Can I push to this branch?